sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloInstanceTest.java

```diff
@@ -82,4 +91,65 @@ public void onChange(final DataChangedEvent dataChangedEvent) {
         assertThat(changeEvent.getValue(), is("value3"));
         assertThat(changeEvent.getChangedType(), is(DataChangedEvent.ChangedType.UPDATED));
     }
+
+    @Test
+    @SneakyThrows
+    public void assertGetWithNonExistentKey() {
+        when(openApiWrapper.getValue(eq("test.nonExistentKey"))).thenReturn(null);
+        assertNull(configCenterRepository.get("/test/nonExistentKey"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void assertUpdate() {
+        final SettableFuture<DataChangedEvent> future = SettableFuture.create();
+        configCenterRepository.watch("/test/children/1", new DataChangedEventListener() {
+
+            @Override
+            public void onChange(final DataChangedEvent dataChangedEvent) {
+                future.set(dataChangedEvent);
+            }
+        });
+        embeddedApollo.addOrModifyProperty("orchestration", "test.children.1", "newValue1");
+        DataChangedEvent changeEvent = future.get(5, TimeUnit.SECONDS);
+        assertThat(changeEvent.getKey(), is("/test/children/1"));
+        assertThat(changeEvent.getValue(), is("newValue1"));
+        assertThat(changeEvent.getChangedType(), is(DataChangedEvent.ChangedType.UPDATED));
+    }
+
+    @Test
+    @SneakyThrows
+    public void assertWatchDeletedChangedType() {
+        final SettableFuture<DataChangedEvent> future = SettableFuture.create();
+        configCenterRepository.watch("/test/children/1", new DataChangedEventListener() {
+
+            @Override
+            public void onChange(final DataChangedEvent dataChangedEvent) {
+                future.set(dataChangedEvent);
+            }
+        });
+        embeddedApollo.deleteProperty("orchestration", "test.children.1");
+        DataChangedEvent changeEvent = future.get(5, TimeUnit.SECONDS);
+        assertThat(changeEvent.getKey(), is("/test/children/1"));
+        assertNull(changeEvent.getValue());
+        assertThat(changeEvent.getChangedType(), is(DataChangedEvent.ChangedType.DELETED));
```

Comment:
now we support delete?

Reply:
we not support   delete ，but can  add dataChangedEventListener